OpenQA/Isotovideo/Utils.pm

```diff
@@ -75,10 +75,17 @@ sub _clone_bare_repo ($clone_url, $clone_depth, $clone_cmd, $cache_dir, $handle_
     $handle_output->($?, qx{$clone_cmd --bare --depth='$clone_depth' '$clone_url' '$cache_dir' 2>&1});
 }
 
-sub _fetch_new_refs ($clone_url, $cache_dir, $branch_args, $handle_output) {
+sub _fetch_new_refs ($clone_url, $cache_dir, $branch_arg, $handle_output) {
     bmwqemu::fctinfo "Updating Git cache for '$clone_url' under '$cache_dir'";
-    $handle_output->($?, qx{git -C "$cache_dir" fetch origin $branch_args 2>&1 2>&1});
-    $handle_output->($?, qx{git -C "$cache_dir" branch --force $branch_args FETCH_HEAD 2>&1 2>&1}) if $branch_args;
+    if ($branch_arg eq '') {
+        # get default branch (usually "master") from remote repo if $branch_arg is empty
```

Comment:
Minor point but the default branch for new repos is `main` and increasingly likely to be what you end up getting. So I'd mention both or only `main`.

```suggestion
        # get default branch (usually "main" or "master") from remote repo if $branch_arg is empty
```